gitlab/resource_gitlab_repository_file.go

```diff
@@ -0,0 +1,153 @@
+package gitlab
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabRepositoryFile() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabRepositoryFileCreate,
+		Read:   resourceGitlabRepositoryFileRead,
+		Update: resourceGitlabRepositoryFileUpdate,
+		Delete: resourceGitlabRepositoryFileDelete,
+
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"file": {
```

Comment:
Can you change this attribute name to `file_path` so that it [matches the API](https://www.terraform.io/docs/extend/hashicorp-provider-design-principles.html#resource-and-attribute-schema-should-closely-match-the-underlying-api)?